### PR TITLE
docs(components): Update InputEmail docs

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations

Documentation needs to be updated to new pattern. This adds an overview Docs page for InputEmail and a web section.

### Fixed

 
Some web component pages didn't have the Code tab included, so those were also added in this PR. Campground cleanup!

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/Drawer/Web.stories.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default {
   component: Drawer,
   parameters: {
     viewMode: "story",
+    previewTabs: { code: { hidden: false } },
```
Comment (darryltec) [inline docs/components/Drawer/Web.stories.tsx:14]: need to overwrite the imports for this

image

**Review by darryltec — CHANGES_REQUESTED** [review]

File: docs/components/Drawer/Web.stories.tsx (hunk 1 above)
Comment (taylorvnoj) [reply docs/components/Drawer/Web.stories.tsx:14]: Ooops! That turned into a waterfall of missing imports 😓  Code tab is working now.

Comment (taylorvnoj) [reply docs/components/Drawer/Web.stories.tsx:14]: Wait, I know Michael is changing things. Disregard the re-request for review for now!

Comment (taylorvnoj) [reply docs/components/Drawer/Web.stories.tsx]: Boom. Done.

File: docs/components/Drawer/Web.stories.tsx [hunk 2]
```diff
@@ -11,6 +11,14 @@ export default {
   component: Drawer,
   parameters: {
     viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@jobber/components/Drawer": ["DrawerGrid"],
```
Comment (darryltec) [inline docs/components/Drawer/Web.stories.tsx:18]: you technicall could add it like 

```suggestion
          "@jobber/components/Drawer": ["Drawer", "DrawerGrid"],
```

if you want it to be a clean import

**Review by darryltec — APPROVED** [review]
1 super minor thing that we could fix or nah. we good tho

**Comment by taylorvnoj** [comment]
The answer is always fix it. Thanks Darryl!

**Pull request merged**